Gems/AWSClientAuth/gem.json

```diff
@@ -14,7 +14,6 @@
         "SDK"
     ],
     "icon_path": "preview.png",
-    "requirements": "",
```

Comment:
nit: why remove this? Most gems.json have this set as an empty string.

Reply:
if this field is expected to be a description - should we list the AWSCore gem as a requirement for this and the AWSMetrics gem?

Reply:
dependencies are defined in "dependencies" field.
remove this to keep file clean if we have nothing to provide in this field.